src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -193,4 +193,20 @@ public void waitForOtherOptionTextToBeClickable(int qnNumber, int responseNumber
         waitForElementToBeClickable(element);
     }
 
+    public void verifyVisibilityAndCloseMoreInfoAboutEqualShareModal() {
+        WebElement moreInfoAboutEqualShareModalLink = browser.driver.findElement(By.id("more-info-equal-share-modal-link"));
+        click(moreInfoAboutEqualShareModalLink);
+        WebElement moreInfoAboutEqualShareModal = browser.driver.findElement(By.id("more-info-equal-share-modal"));
+        waitForElementVisibility(moreInfoAboutEqualShareModal);
+        closeMoreInfoAboutEqualShareModal();
+    }
+
+    public void closeMoreInfoAboutEqualShareModal() {
+        WebElement closeButton = browser.driver.findElement(By
+                .xpath("//div[@id='more-info-equal-share-modal']//button[@class='close']"));
+        waitForElementToBeClickable(closeButton);
+        click(closeButton);
+        waitForElementToDisappear(By.id("more-info-equal-share-modal"));
+    }
```

Comment:
@chowyb looks like this page uses a lot of 'locate element on the spot' kind of code whereas the cleaner way is to use `@FindBy` annotations such as done [here](https://github.com/TEAMMATES/teammates/blob/master/src/test/java/teammates/test/pageobjects/HomePage.java). Not sure if it is possible though because xpath for some elements in this page are not fixed. We should explore Selenium's _Page Object Model_ a bit more ([example](http://toolsqa.com/selenium-webdriver/page-object-pattern-model-page-factory/)) to see if we can improve the status quo of this page and other similar pages.